### ISSUE TITLE
Problems with image scrubber + zooming

### ISSUE THREAD

**Issue by rabernat** [holoviz/hvplot#543]
#### ALL software version info
(this library, plus any other relevant software, e.g. bokeh, python, notebook, OS, browser, etc)

- python 3.8
- hvplot 0.6.0
- holoviews 1.13.5
- jupyterlab 2.2.9
- xarray 0.16.1
- Chrome 87.0.4280.88

#### Description of expected behavior and the observed behavior

I am making a "scrubber" style map. When I zoom in and then use the scrubber, the image resets the zoom. This makes it impossible to animate a small region.

https://user-images.githubusercontent.com/1197350/103845368-ec3ad980-5069-11eb-9c55-3e2491d3ec1f.mov

**This definitely used to work** in some recent iteration of this complex software stack. As a long time holoviews user, I find this sort of regression quite frustrating and unfortunately impossible for me to debug on my own.

#### Complete, minimal, self-contained example code that reproduces the issue

This example is from Pangeo Gallery and can be run via the binder links on this page: http://gallery.pangeo.io/repos/pangeo-gallery/physical-oceanography/01_sea-surface-height.html
I have been using this example for years to demonstrate the power of holoviews to many different audiences.

```python
import hvplot.xarray
from intake import open_catalog
cat = open_catalog("https://raw.githubusercontent.com/pangeo-data/pangeo-datastore/master/intake-catalogs/ocean.yaml")
ds  = cat["sea_surface_height"].to_dask()
ds.sla.hvplot.image('longitude', 'latitude',
                    rasterize=True, dynamic=True, width=800, height=450, 
                    widget_type='scrubber', widget_location='bottom', cmap='RdBu_r')
```

Thanks to all the maintainers for your work on this amazing open source tool.

**Comment by rabernat** [comment]
Fixed by upgrading to holoviews 1.14.0. Sorry for the noise.

**Comment by philippjfr** [comment]
Good to hear!
